README.md

```diff
@@ -6,3 +6,4 @@ Contributors:
 | Name | GitHub Username | Portfolio URL |
 | ---- | --------------- | ------------- |
 | Elsa Culler | eculler | https://eculler.github.io/ |
+| Peter Kobylarz | peterkobylarz | https://peterkobylarz.github.io/
```

Comment:
Can you add a pipe | to the end of line 9?

Reply:
I've added the pipe and re-committed the change. Thanks!